kolibri/content/test/test_annotation.py

```diff
@@ -61,13 +61,13 @@ def setUp(self):
 
     def test_all_content_nodes_available(self):
         ContentNode.objects.exclude(kind=content_kinds.TOPIC).update(available=True)
-        recurse_availability_up_tree()
+        recurse_availability_up_tree(channel_id="6199dde695db4ee4ab392222d5af1e5c")
         self.assertTrue(ContentNode.objects.get(id="da7ecc42e62553eebc8121242746e88a").available)
         self.assertTrue(ContentNode.objects.get(id="2e8bac07947855369fe2d77642dfc870").available)
 
     def test_no_content_nodes_available(self):
         ContentNode.objects.filter(kind=content_kinds.TOPIC).update(available=True)
-        recurse_availability_up_tree()
+        recurse_availability_up_tree(channel_id="6199dde695db4ee4ab392222d5af1e5c")
```

Comment:
Need to update the next test below too, `def test_one_content_node_available(self):`

Reply:
Change it!